util/deephash/deephash.go

```diff
@@ -452,46 +437,22 @@ func (h *hasher) hashAddrv(v addressableValue) {
 }
 
 func makeMemHasher(n uintptr) typeHasherFunc {
-	return func(h *hasher, v addressableValue) {
-		h.HashBytes(unsafe.Slice((*byte)(v.Addr().UnsafePointer()), n))
-	}
-}
-
-// hashSliceMem hashes v, of kind Slice, with a memhash-able element type.
-func (h *hasher) hashSliceMem(v addressableValue) {
```

Comment:
`hashSliceMem` is deleted because we inline where it is used in `genTypeHasher` so that we can access the element size from the type.